packages/formats/test/index.test.js

```diff
@@ -68,20 +88,28 @@ describe('@rdfjs-elements/formats-pretty', () => {
       for (const format of Object.values(mediaTypes)) {
         it(`graph ${file} in format ${format}`, async () => {
           // given
-          const graph = await $rdf
+          const data = await $rdf
             .dataset()
             .import($rdf.fromFile(join(__dirname, `graphs/${file}`)))
 
           // when
           const serialized = await getStream(
-            serializers.import(format, graph.toStream())
+            serializers.import(format, data.toStream())
           )
-          const roundTrip = await $rdf
-            .dataset()
-            .import(parsers.import(format, toStream(serialized)))
+          const parserStream = parsers
+            .import(format, toStream(serialized))
+            .pipe(
+              new Transform({
+                objectMode: true,
+                transform({ subject, predicate, object, graph }, _, callback) {
+                  callback(null, $rdf.quad(subject, predicate, object, graph))
+                },
+              })
+            )
```

Comment:
Annoyingly, it turns out that the Quads produced by n3 are not working well with `rdf-canonize`. Feels like this will bite us at some point. Maybe the terms should be repackaged using `@rdfjs/data-model` in the n3 parser stream?